lib/src/component/dom_components.dart

```diff
@@ -87,451 +110,451 @@ abstract class Dom {
   /// Returns a new builder that renders an `<a>` tag with getters/setters for all DOM-related React props,
 
   /// optionally backed by a specified map.
-  static DomProps a([Map backingMap]) => DomProps(react.a, backingMap);
+  static DomProps a([backingMap]) => $domPropsConfig.map(backingMap)..componentFactory = react.a;
```

Comment:
From here on down, the only change is refactoring Dom components to use the new config class.